Neos.Flow/Classes/Mvc/Controller/AbstractController.php

```diff
@@ -338,12 +341,15 @@ protected function redirectToUri($uri, $delay = 0, $statusCode = 303)
      *
      * NOTE: This method only supports web requests and will throw an exception if used with other request types.
      *
+     * TODO: statusMessage argument is deprecated and should be removed in next major.
+     *
      * @param integer $statusCode The HTTP status code
      * @param string $statusMessage A custom HTTP status message
      * @param string $content Body content which further explains the status
      * @throws UnsupportedRequestTypeException If the request is not a web request
      * @throws StopActionException
      * @api
+     * @deprecated This method will no longer accept a status message after next major, but still exist.
```

Comment:
According to the [spec](https://www.w3.org/Protocols/rfc2616/rfc2616-sec6.html) the status message `...MAY be replaced by local equivalents without affecting the protocol` so I don't think that we should change the signature here (especially because it would be a very nasty breaking change because the message would suddenly get the response content

Reply:
I will remove the comment, BUT that would mean we keep the unused param until eternity?